lib/coinbase/client/api/webhooks_api.rb

```diff
@@ -0,0 +1,286 @@
+=begin
+#Coinbase Platform API
+
+#This is the OpenAPI 3.0 specification for the Coinbase Platform APIs, used in conjunction with the Coinbase Platform SDKs.
+
+The version of the OpenAPI document: 0.0.1-alpha
+Contact: yuga.cohler@coinbase.com
+Generated by: https://openapi-generator.tech
+Generator version: 7.7.0
```

Comment:
just want to flag this as my generator is at 7.7.0, pls let me know if there's any issue with this